packages/testing-helpers/src/litFixture.js

```diff
@@ -1,25 +1,49 @@
+import { TemplateResult } from 'lit-html';
 import { fixtureWrapper } from './fixtureWrapper.js';
 import { render } from './lit-html.js';
 import { elementUpdated } from './elementUpdated.js';
 
+/**
+ * @typedef {
+     import('lit-html').TemplateResult
+   | import('lit-html').TemplateResult[]
+   | Node
+   | Node[]
+   | string
+   | string[]
+   | number
+   | number[]
+   | boolean
+   | boolean[]
+ } LitHTMLRenderable
+ */
+
+const isCommentNode = ({ nodeType }) => nodeType === 8;
+
+const not = p => x => !p(x);
+
 /**
  * Setups an element synchronously from the provided lit-html template and puts it in the DOM.
  *
  * @template {Element} T - Is an element or a node
- * @param {import('lit-html').TemplateResult} template
+ * @param {LitHTMLRenderable} template
  * @returns {T}
  */
 export function litFixtureSync(template) {
   const wrapper = fixtureWrapper();
   render(template, wrapper);
-  return /** @type {T} */ (wrapper.children[0]);
+  if (template instanceof TemplateResult) {
+    return /** @type {T} */ (wrapper.firstElementChild);
+  }
+  const [node] = Array.from(wrapper.childNodes).filter(not(isCommentNode));
```

Comment:
Should we also filter text nodes with no or only whitespace chars? This can happen when indenting templates

Reply:
I pushed some code for this but I don't understand the test case well enough to write a test yet.

Reply:
This creates some empty text nodes:

```fixture(html`

<div></div>

`)```

where I expect to get the div from the fixture

Reply:
Ok I added a test for this, but control won't get to `wrapper.childNodes` when a `TemplateResult` is passed:

```js
if (template instanceof TemplateResult) {
  return /** @type {T} */ (wrapper.firstElementChild);
}
```
Is this a concern when passing Node or String?

WRT string also, since lit-html's `render` now supports string as well, should we consider removing `stringFixture` altogether?

Reply:
Why would we make this logic different between `TemplateResult` and others? It's just the means of defining the template, right? 

Indeed we can remove `stringFixture` but that's breaking so for now it can just forward to the regular fixture. stringFixture might be nice if you don't want to use lit-html, but that's already imported in those files I think.

Reply:
The logic is different because the type is different.

Consider 

```js
render(document.createTextNode('not an element'))
```

In that case, 

```js
wrapper.firstElementChild; // null
```

However, `wrapper.childNodes.length` is 3 - a comment node, our text node, and another comment node.

Reply:
Right, I see. Thanks for the explanation.